scripts/plot_results.py

```diff
@@ -18,7 +18,7 @@ def power_by_carrier(n):
         n.generators.carrier).sum().T 
     
     if not n.storage_units.empty:
-        sto = n.storage_units_t.p.T.groupby(
+        sto = n.storage_units_t.p.T.clip(lower=0).groupby(
```

Comment:
Is there an artifact that shows up in the plot unless you do this?

Reply:
This is the change that allows the plot to show just discharge (instead of the net of charge and discharge, which is what it was showing previously).